acp/common.php

```diff
@@ -41,97 +41,97 @@
-	'DIGESTS_EXECUTION_TIME'								=> 'Temps d’exécution (sec)',
+	'DIGESTS_EXCLUDE_FORUMS_EXPLAIN'						=> 'Saisissez le forum_id des forums qui ne doivent pas figurer dans le résumé. Séparez les forum_ids par des virgules. En indiquant 0, aucun forum ne sera exclu. Le forum_id d’un forum correspond à la valeur du paramètre « f » dans le champ de l’URL. Exemple : http://www.example.com/phpBB3/viewforum.php?f=1. Ne pas utiliser de forum_ids qui correspondent à des catégories. <i>Ce réglage est ignoré si l’option « Sujets cochés seulement » est activée par l’utilisateur.</i>',
+	'DIGESTS_EXECUTION_TIME'								=> 'Temps d’exécution (secondes)',
 	'DIGESTS_EXPAND'										=> 'Agrandir les paramètres du résumé de l’utilisateur',
 	'DIGESTS_FREQUENCY_EXPLAIN'								=> 'Les résumés hebdomadaires sont envoyés le jour de la semaine défini dans les réglages généraux de Digests. Les résumés mensuels sont envoyés le premier du mois. Le Temps Universel Coordonné (UTC) est utilisé pour déterminer le jour de la semaine.',
 	'DIGESTS_FORMAT_FOOTER' 								=> 'Format du résumé',
 	'DIGESTS_FROM_EMAIL_ADDRESS'							=> 'Adresse électronique utilisée pour expédier les résumés',
-	'DIGESTS_FROM_EMAIL_ADDRESS_EXPLAIN'					=> 'Lorsque les utilisateurs recoivent un résumé, cette adresse électronique va s’afficher dans le champ « De ». S’il est laissé vide, l’adresse électronique de votre Forum sera utilisé par défaut. Faites attention lorsque vous utilisez une adresse électronique dont le domaine ne correspond pas à celui sur lequel est hébergé votre Forum, dans la mesure où votre serveur de courriels ou le serveur de courriels des utilisateurs pourra interprêter le courriel comme étant du spam.',
+	'DIGESTS_FROM_EMAIL_ADDRESS_EXPLAIN'					=> 'Lorsque les utilisateurs recoivent un résumé, cette adresse électronique va s’afficher dans le champ « De ». S’il est laissé vide, l’adresse électronique du forum sera utilisée par défaut. Faites attention lorsque vous utilisez une adresse électronique dont le domaine ne correspond pas à celui sur lequel est hébergé le forum, car votre serveur de courriels ou ceux des utilisateurs pourraient interpréter le courriel comme étant indésirable.',
 	'DIGESTS_FROM_EMAIL_NAME'								=> 'Nom de l’expéditeur du courriel',
-	'DIGESTS_FROM_EMAIL_NAME_EXPLAIN'						=> 'C’est le nom qui apparaîtra en clair dans le champ « De » du logiciel de courriel client de l’utilisateur. Si le champ est laissé vide il s’identifera en tant que robot de résumé de votre forum.',
+	'DIGESTS_FROM_EMAIL_NAME_EXPLAIN'						=> 'C’est le nom qui apparaîtra en clair dans le champ « De » du logiciel de courriel client de l’utilisateur. Si le champ est laissé vide, il s’identifiera en tant que robot de résumé du forum.',
```

Comment:
> Si le champ est laissé vide, il s’identifiera en tant que robot de résumé du forum.',

Que ce soit en français ou en anglais, ça n'a aucun sens. 😮  
En plus il y a un bug au niveau du code si l'auteur de l'extension veut faire ce qu'il dit. 🤔 

En vrai, si le champ est laissé vide, alors ca affichera le nom indiqué dans "Contact du forum".  
Sauf que dans les faits l'auteur récupérer l'adresse mail du contact du forum et non le nom.

Donc, soit on traduit en se basant sur ce qu'est supposer faire l'option sans se soucier du bug.  
Soit on traduit avec ce que fait réellement l'extension.

Peut-être voir à reporter le bug.

Reply:
![IMG_ 2022-10-29 at 17 32 53](https://user-images.githubusercontent.com/87082648/198840191-b8492e38-4ae8-4563-8de6-8596b4675cda.png)

Reply:
La capture d'écran que t'as posté n'est pas en lien avec cette clé de langue.  

Je laisse cette conversation ouverte, juste pour garder en mémoire qu'il y a le bug. Mais en soit pour la traduction on peut passer outre (pour le moment)

Reply:
Pourtant on parle bien de la ligne 101 non ?
`This is the plain text FROM name that will appear in the email client. If left blank it will identify itself as a digests robot for your board.`
Bon, j'ai vu que tu étais intervenu, merci. 
J'attends tes instructions pour la suite.

Reply:
Au temps pour moi.
Je me suis emmêlé les pinceaux avec l'autre où il est question du "REPLY_TO"